.gitignore

```diff
@@ -29,4 +29,7 @@ yarn-error.log*
 # Optional eslint cache
 .eslintcache
 
-vite.config.ts.timestamp-*
\ No newline at end of file
+vite.config.ts.timestamp-*
+
+# fly.io
+fly.toml
```

Comment:
Did you mean to include this `fly.toml` exclusion?

Reply:
to prevent myself from accidentally committing mine to this PR ;)